### ISSUE TITLE
Strange `%` selectors using GitHub

### ISSUE THREAD

**Issue by peterbe** [peterbe/minimalcss#77]
See...
```
▶ node --trace-warnings bin/minimalcss.js -d --verbose https://github.com/peterbe/minimalcss/pull/74
0% SyntaxError: Unmatched selector: %
50% SyntaxError: Unmatched selector: %
100% SyntaxError: Unmatched selector: %
10% SyntaxError: Unmatched selector: %
30% SyntaxError: Unmatched selector: %
70% SyntaxError: Unmatched selector: %
7% SyntaxError: Unmatched selector: %
80% SyntaxError: Unmatched selector: %
3% SyntaxError: Unmatched selector: %
20% SyntaxError: Unmatched selector: %
23% SyntaxError: Unmatched selector: %
40% SyntaxError: Unmatched selector: %
43% SyntaxError: Unmatched selector: %
60% SyntaxError: Unmatched selector: %
63% SyntaxError: Unmatched selector: %
90% SyntaxError: Unmatched selector: %
93% SyntaxError: Unmatched selector: %
/*
Generated 2018-01-18T01:46:30.717Z by minimalcss.
Took 7.46 seconds to generate 32.04 KB of CSS.
Based on 3 stylesheets totalling 554.38 KB.
...
```

**Comment by stereobooster** [comment]
can be closed in favour of https://github.com/peterbe/minimalcss/issues/79

**Comment by peterbe** [comment]
> can be closed in favour of #79

Yea. My bad for not remembering I had already filed.
